src/Components/CustomSeparator.jsx

```diff
@@ -20,6 +20,12 @@ navMap.set("/license/generate", {
   title: "Generate Product Key"
 });
 
+navMap.set("/license/search", {
```

Comment:
Try to refactor navMap and add it in routes.js. Export it from there and use it because it is duplicated in two components.

Reply:
Done